setup.py

```diff
@@ -15,19 +16,21 @@
     from distutils.core import Extension
     from distutils.core import setup
 from distutils.command.build_ext import build_ext
-from distutils import errors
 from distutils import dep_util
 from distutils import log
 
 
+IS_PYTHON3 = sys.version_info[0] == 3
+CIBUILDWHEEL = os.environ.get('CIBUILDWHEEL', '0') == '1'
```

Comment:
why limit py_limited_api support to cibuildwheel only? I understand this is currently the mainstream approach to wheel building but what's special about it for the sake of building abi3 wheels that we need to check for `CIBUILDWHEEL` environment variable being set?
Can we enable this build option via a different environment variable that doesn't tie it too closely to that specific build environment?

Reply:
The limited api is less performant on the python api.
A user that would build brotli by himself would most likely only build it for himself for his version, not needing cross version compatibility, so the user could make use of version specific optimisations.
The optimisations are just minor and are mainly just smarter function calls with less jumps overall.

But sure, we can rename the setting and let cibuildwheel apply it then.
I guess that ``PYTHON3_USE_LIMITED_API`` would work?

Reply:
I suggest to prefix with `BROTLI_`, something like `BROTLI_USE_PYTHON3_LIMITED_API`

Reply:
Done,
as reminder, the cibuildwheel runner now has to set ``BROTLI_PYTHON3_LIMITED_API`` to 1.